lib/doorkeeper/models/access_token_mixin.rb

```diff
@@ -435,6 +435,10 @@ def attributes_for_token_generator
         if Doorkeeper.config.polymorphic_resource_owner?
           attributes[:resource_owner] = resource_owner
         end
+
+        Doorkeeper.config.custom_access_token_attributes.each do |attribute_name|
+          attributes[attribute_name] = send(attribute_name)
```

Comment:
can we at least use `public_send` as bot suggested?

Reply:
Yes, done!